imnodes.h

```diff
@@ -127,9 +128,11 @@ void Link(int id, int start_attr, int end_attr);
 // Set's the node's position corresponding to the node id. You can even set the
 // position before the node has been created with BeginNode().
 void SetNodePos(int node_id, const ImVec2& pos);
+void SetNodeOriginPos(int node_id, const ImVec2& origin_pos);
```

Comment:
Like I already mentioned, this should be named `SetNodeGridSpacePos` 🙂

Reply:
Cool, I will update the PR.

Reply:
Done